src/tests/test_stack_switching.py

```diff
@@ -627,12 +628,14 @@ def test_memory_leak(selenium, script):
         """
         """
         const t = pyodide.globals.get("test");
-        let p = [];
-        // warm up first to avoid edge problems
-        for (let i = 0; i < 200; i++) {
-            p.push(t.callPromising(1));
+        for (let i = 0; i < 1; i++) {
```

Comment:
What's this one-time loop for? If you wanted to limit the scope, just wrapping them with `{ ... }` would work I guess,

Reply:
Well this test is currently broken anyways. It looks like a possible v8 bug? It's probably going to be a lot of effort to track down...